src/index.html

```diff
@@ -7,8 +7,93 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        alt="logo"
+        href="#Home"
+        class="logo"
+        src="images/logo.png">
```

Comment:
```suggestion
      <img
        alt="logo"
        href="#Home"
        class="logo"
        src="images/logo.png"
      >
```

Reply:
Not fixed 
<img width="730" alt="image" src="https://user-images.githubusercontent.com/99403198/219866226-80f04a44-d0e3-4eb1-9703-edf5f5a16da6.png">

Reply:
Not fixed